R/aaa_survival_prop.R

```diff
@@ -138,16 +138,17 @@ cph_survival_pre <- function(new_data, object) {
 #' @param new_data Data for prediction
 #' @param .times A vector of integers for prediction times.
 #' @param output One of "surv" or "haz".
+#' @param training_data A list of `x` and `y` containing the training data.
 #' @param ... Options to pass to [survival::survfit()]
 #' @return A nested tibble
 #' @keywords internal
 #' @export
-coxnet_survival_prob <- function(x, new_data, .times, output = "surv", ...) {
+coxnet_survival_prob <- function(x, new_data, .times, output = "surv", training_data, ...) {
```

Comment:
```suggestion
coxnet_survival_prob <- function(x, new_data, .times, training_data, output = "surv", ...) {
```

Since `training_data` is required, and `output` is optional, you might consider switching the order here
https://design.tidyverse.org/args-data-details.html